documentation/Generate-code-from-TypeSpec.md

```diff
@@ -0,0 +1,113 @@
+# Before you Start
+
+[TypeScript Azure SDK Design Guidelines](https://azure.github.io/azure-sdk/typescript_introduction.html) is the overall design guideline of the client SDK.
+Make sure you are familiar with concepts such as "Service Client" and "Packaging".
+
+Make sure you are familiar with Git.
+
+See [Prerequisite](https://www.npmjs.com/package/@azure-tools/typespec-ts) of `@azure-tools/typespec-ts`. Make sure that NodeJS, JDK is installed.
+
+# Generate SDK
+
+## Use TypeSpec defined in REST API specifications
+
+:ballot_box_with_check: The package structure in the azure-rest-api-specs repository should follow [these guidelines](https://github.com/Azure/azure-rest-api-specs/blob/main/documentation/typespec-structure-guidelines.md#structure).
+
+Here is a sample for [tspconfig.yaml](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/contosowidgetmanager/Contoso.Management/tspconfig.yaml).
```

Comment:
Can we provide 3 examples, for HLC, RLC and modular client?

Reply:
hlc doesn't use typespec, I think we can just provide RLC and Modular examples